src/commands/chain/proof.ts

```diff
@@ -20,7 +20,7 @@ import { BaseCommand, baseFlags } from "../../baseCommand.js";
 import { peerIdToUint8Array } from "../../lib/chain/conversions.js";
 import { setChainFlags, chainFlags } from "../../lib/chainFlags.js";
 import { CHAIN_FLAGS, PRIV_KEY_FLAG_NAME } from "../../lib/const.js";
-import { getDealClient, signBatch } from "../../lib/dealClient.js";
+import { getDealClient, signBatch, populate } from "../../lib/dealClient.js";
```

Comment:
`populate` sounds like a too generic of a name for a public API. No way to tell what it does by its name :(

Reply:
maybe `populateTx`?

Reply:
I wonder where it comes from, cannot find it in the fluencelabs/deal 🤔 

Also, is it OK that you import `.js` and not typed `.ts`?

Reply:
it comes from CLI. You can see it defined in this very PR.

using js extension is what I am doing for a long time. It's a eslint rule that enforced a while back cause there were some problems that I already do not remember that went away if you use it this way instead of using no-extension/ts-extsion

Reply:
Thank you for the explanation!